packages/react/src/file-input/FileInput.tsx

```diff
@@ -67,18 +68,22 @@ export const FileInput = forwardRef<HTMLInputElement, FileInputProps>(
 			[onChangeProp]
 		);
 
+		const fallbackHint =
+			hint || (accept && `Accepted files include: ${accept}`);
```

Comment:
Can you parse them like FileUpload does to meaningful extensions?

Reply:
I did look into this but unfortunately we've used the native `accept` type rather than the `react-dropzone` ones. So it accepts a `string` and modifying it would be a breaking change.

Reply:
Can't we normalise, split it and map it similar to what we do in L174 of FileUpload?

Reply:
And is there anything stopping us from also supporting an array?

Reply:
>And is there anything stopping us from also supporting an array?

Yeah, that might be the trick! It could be a tuple based on the `react-dropzone` types. String coercion would be complex and prone to issues, I don't think it's the right tool but an array is. I'll put something together.

- [ ] tuple for accept